src/compiler/types.ts

```diff
@@ -4017,8 +4017,14 @@ namespace ts {
     export enum DiagnosticCategory {
         Warning,
         Error,
+        Info,

```

Comment:
I would call it `Suggession`. `Info` and `Message` are pretty much the same thing.

Reply:
Suggestion

Reply:
FWIW, Roslyn calls it `Info`.  What *is* the difference between `Info` and `Message`?  Is `Message` actually `Telemetry`?

Reply:
Message is like description of a quick fix. user never sees that as a Diagnostic really.

Reply:
This isn't quite what Roslyn calls `Info`, so I'm fine with either `Info` or `Suggestion` (or `Hint`, which appears to be the VS Code name).

Reply:
I'd rather be consistent, but if we plan to have more-specific categories than `Info` in the future, `Suggestion` is fine with me.